service/controller/clusterapi/v22/version_bundle.go

```diff
@@ -39,7 +39,8 @@ func VersionBundle() versionbundle.Bundle {
 				Version: "0.4.1",
 			},
 		},
-		Name:    "cluster-operator",
-		Version: "0.22.0",
+		Name:     "cluster-operator",
+		Provider: p,
```

Comment:
I discussed with Tuomas to use the provider here. For the time being this should do the trick.